R/score_plates.R

```diff
@@ -250,7 +250,9 @@ optimize_multi_plate_design <- function(batch_container, across_plates_variables
 
   if (!is.null(within_plate_variables)) {
     plate_levels <- unique(bc$get_locations()[[plate]])
-    scoring_funcs <- purrr::map(within_plate_variables, ~ mk_plate_scoring_functions(bc, plate = plate, row = row, column = column, group = .x)) |>
+    scoring_funcs <- purrr::map(within_plate_variables,
```

Comment:
i'm not sure if it's a styler issue, or the line just too long. maybe move `within_plate_variables` on the next line before running styler.

Reply:
Styled!